canonical-web-and-design/hiring.canonical.com
# Add category to job filter

## Done
Jobs in the applications review pending chart's filter are categorized by departments.

## QA
- `dotrun`
- Open applications review page
- Click on search and filter component to open it.
- Verify jobs are categorized by departments
